.github/actions/spelling/expect.txt

```diff
@@ -1,180 +0,0 @@
-abcdef
```

Comment:
What was expect before is now split into Lombiq.common.txt and OSOCE's own allow.txt. Actually, since we've now settled on other projects excluding submodules (that are also in OSOCE) from spell checking, I'll split OSOCE's allow.txt into an allow.txt and expect.txt (it was like that for some time in this PR). Docs also to be updated (same as https://github.com/Lombiq/GitHub-Actions/pull/203#discussion_r1112285185).